code/modules/food_and_drinks/food/snacks/meat.dm

```diff
@@ -340,6 +340,19 @@
 	icon_state = "shadowmeat"
 	desc = "It is covered in a strange darkness. This slab's magical properties appear to be drastically weakened due to the synthetic nature of the meat."
 
+/obj/item/reagent_containers/food/snacks/meat/slab/plagued
+	name = "meat"
+	desc = "A slab of meat. Eating it will be a VERY GOOD idea(actually not)."
+	icon_state = "meat"
+	dried_type = /obj/item/reagent_containers/food/snacks/sosjerky/
+	bitesize = 3
+	list_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/cooking_oil = 2, /datum/reagent/plaguebacteria = 3) //It is infected by plague
+	slice_path = /obj/item/reagent_containers/food/snacks/meat/raw_cutlet/plain
```

Comment:
You should probably make a raw_cutlet/plain/plague obj to give more differentiation, unless the intention is for plague meat to be indistinguishable from regular cutlets once sliced up?